File: app/src/main/java/com/marekguran/tzi/DynamicColors.java
package com.marekguran.tzi;

import android.app.Application;

public class DynamicColors extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        com.google.android.material.color.DynamicColors.applyToActivitiesIfAvailable(this);
    }
}

File: app/src/main/java/com/marekguran/tzi/DynamicColors.kt
package com.marekguran.tzi

import android.app.Application
import com.google.android.material.color.DynamicColors

class DynamicColors : Application() {
    override fun onCreate() {
        super.onCreate()
        DynamicColors.applyToActivitiesIfAvailable(this)
    }
}